File: project/yeonwoo-onboarding/src/main/java/com/innercircle/yeonwoo_onboarding/repository/SurveyResponseRepository.java
package com.innercircle.yeonwoo_onboarding.repository;

import com.innercircle.yeonwoo_onboarding.domain.SurveyResponse;
import org.springframework.data.jpa.repository.JpaRepository;
import java.util.List;

public interface SurveyResponseRepository extends JpaRepository<SurveyResponse, String> {
    List<SurveyResponse> findBySurveyId(String surveyId);
}

File: project/taehwan-onboarding/src/main/kotlin/org/survey/taehwanonboarding/domain/repository/survey/SurveyResponseRepository.kt
package org.survey.taehwanonboarding.domain.repository.survey

import org.springframework.data.jpa.repository.JpaRepository
import org.survey.taehwanonboarding.domain.entity.response.SurveyResponse

interface SurveyResponseRepository : JpaRepository<SurveyResponse, Long> {
    fun findAllBySurveyId(surveyId: Long): List<SurveyResponse>
}